ballerina/googlesheets_client.bal

```diff
@@ -31,21 +31,21 @@ type RowValues record {
 
 # The client used by the generated persist clients to abstract and
 # execute API calls that are required to perform CRUD operations.
-public client class GoogleSheetsClient {
+public isolated client class GoogleSheetsClient {
 
     private final sheets:Client googleSheetClient;
     private final http:Client httpClient;
-    private string spreadsheetId;
-    private int sheetId;
-    private string entityName;
-    private string tableName;
-    private string range;
-    private map<SheetFieldMetadata> fieldMetadata;
-    private map<string> dataTypes;
-    private string[] keyFields;
-    private function (string[]) returns stream<record {}, Error?>|Error query;
-    private function (anydata) returns record {}|NotFoundError queryOne;
-    private map<function (record {}, string[]) returns record {}[]|error> associationsMethods;
+    private final string & readonly spreadsheetId;
+    private final int & readonly sheetId;
+    private final string & readonly entityName;
+    private final string & readonly tableName;
+    private final string & readonly range;
+    private final map<SheetFieldMetadata> & readonly fieldMetadata;
+    private final map<string> & readonly dataTypes;
+    private final string[] & readonly keyFields;
+    private final (function (string[]) returns stream<record {}, Error?>|Error) & readonly query;
+    private final (function (anydata) returns record {}|NotFoundError) & readonly queryOne;
```

Comment:
To call this in L135 you need to make this isolated with https://github.com/ballerina-platform/ballerina-lang/pull/40410

```suggestion
    private isolated function (anydata) returns record {}|NotFoundError queryOne;
```

Please check other errors too.

Reply:
updated in [c09e03c](https://github.com/ballerina-platform/module-ballerina-persist/pull/214/commits/c09e03c0172c60d0ca01d6f6856daacb8acd19f2)